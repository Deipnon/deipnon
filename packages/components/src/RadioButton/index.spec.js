import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-emotion'

import Radio from '.'

describe('Radio', () => {
	test('renders correctly', () => {
		const tree = renderer.create(<Radio checked="false" value="test" name="test">Radio Button</Radio>).toJSON()
		expect(tree).toMatchSnapshot()
	})
})
