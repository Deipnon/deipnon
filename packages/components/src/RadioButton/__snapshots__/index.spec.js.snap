// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio renders correctly 1`] = `
<label
  className="css-nfz16p"
>
  <input
    checked="false"
    className="css-1pmzrhc"
    type="radio"
  />
  Radio Button
</label>
`;
